### DISCUSSION TITLE
lesson 7 importing the chainlink contract

### DISCUSSION THREAD

**Discussion by kapasainitishreddy** [smartcontractkit/full-blockchain-solidity-course-py#1679] [Q&A] [answered]
CompilerError: solc returned the following errors:

ParserError: Source "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" not found: File not found. Searched the following locations: "".
 --> contracts/lottery.sol:3:1:
  |
3 | import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";

**Comment by cromewar** [comment, marked as answer]
Hello @NitishReddyKapa 
Be sure to use the correct chainlink version, the one used on the contracts are for solidity 0.6, if you want to implement it with a newer version be aware some contract will have different names, also the VRF service has changed a lot since the video, so I would recommend to keep using V2 and sol 0.6.

**Reply by kapasainitishreddy** [reply]
thank you @cromewar

**Reply by Brandoneos** [reply]
I was wondering if this was still possible because there is a new VRF v2 contract here: https://docs.chain.link/docs/vrf/v2/subscription/examples/get-a-random-number/

And it seems the version that I should use is 0.8.0 at least not 0.6.0. So then I'm having trouble matching it up with the contracts in the course. Is there a solution to this or am I getting something wrong here?

**Reply by cromewar** [reply]
Hello @Brandoneos 
You can still use the older version but be sure to change the repo version on github for the one used on the course.
